lexers/internal/api.go

```diff
@@ -104,11 +102,17 @@ func Match(filename string) chroma.Lexer {
 	for _, lexer := range Registry.Lexers {
 		config := lexer.Config()
 		for _, glob := range config.Filenames {
-			if fnmatch.Match(glob, filename, 0) {
+			ok, err := filepath.Match(glob, filename)
+			if err != nil {
+				panic(err)
```

Comment:
Can you add a test that imports `lexers/all` and calls this function - this will exercise the glob compilation to ensure there are no errors for users at runtime.